target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -27,6 +27,12 @@
 
 	chosen {
 		linux,stdout-path = "serial0:115200n8";
+		leds {
+			boot = &power;
+			failsafe = &ledgnr;
+			running = &power;
+			upgrade= &ledgnr;
+		};
 	};
```

Comment:
ok, did not realize that. i think this needs to be be changed to aliases globablly. anyhow once the & vs && stuff is fixed i can merge this as is